lib/is_bullshit.rb

```diff
@@ -13,10 +14,12 @@ class Object
     '0' => true,
     'nil' => true,
     'null' => true,
+    Float::INFINITY => true,
+    BigDecimal::INFINITY => true
   }
 
   def bullshit?
-    !!BULLSHIT_VALUES[self]
+    !!BULLSHIT_VALUES[self] or (self.nan? if [Float, BigDecimal].member? self.class)
```

Comment:
Do you have an example? It works w/ `nil` and passes the tests. Can `null`be even an object in ruby??